apps/mobile/src/screens/Login/EnterEmailScreen.tsx

```diff
@@ -0,0 +1,93 @@
+import { useNavigation } from '@react-navigation/native';
+import { useCallback, useState } from 'react';
+import { KeyboardAvoidingView, View } from 'react-native';
+import { SafeAreaView } from 'react-native-safe-area-context';
+
+import { LoginStackNavigatorProp } from '~/navigation/types';
+
+import { Button } from '../../components/Button';
+import { FadedInput } from '../../components/FadedInput';
+import { IconContainer } from '../../components/IconContainer';
+import { Typography } from '../../components/Typography';
+import { useLogin } from '../../hooks/useLogin';
+import { BackIcon } from '../../icons/BackIcon';
+import { magic } from '../../magic';
+
+export function EnterEmailScreen() {
+  const navigation = useNavigation<LoginStackNavigatorProp>();
+
+  const [email, setEmail] = useState('');
+  const [error, setError] = useState('');
+  const [isLoggingIn, setIsLoggingIn] = useState(false);
+
+  const [login] = useLogin();
+
+  const handleContinue = useCallback(async () => {
+    setError('');
+    setIsLoggingIn(true);
+
+    try {
+      const token = await magic.auth.loginWithMagicLink({ email });
```

Comment:
I think you can pass in `showUI: false` here, which lets us show our own message/ui instead of their default "check your email" screen